src/js/tabs/pid_tuning.js

```diff
@@ -1738,29 +1759,51 @@ pid_tuning.initialize = function (callback) {
             throttleCurve.width = throttleCurve.height *
                 (throttleCurve.clientWidth / throttleCurve.clientHeight);
 
+            const curvescale = throttleLimitType === THROTTLE_LIMIT_TYPES.SCALE ? throttleLimitPercent : 1;
             const canvasHeight = throttleCurve.height;
             const canvasWidth = throttleCurve.width;
 
             // math magic by englishman
+            const topy = canvasHeight * (1 - curvescale);
             const midx = canvasWidth * mid;
             const midxl = midx * 0.5;
             const midxr = (((canvasWidth - midx) * 0.5) + midx);
-            const midy = canvasHeight - (midx * (canvasHeight / canvasWidth));
-            const midyl = canvasHeight - ((canvasHeight - midy) * 0.5 *(expo + 1));
-            const midyr = (midy / 2) * (expo + 1);
+            const midy = (canvasHeight - curvescale * (midx * (canvasHeight / canvasWidth)));
```

Comment:
// math magic by englishman :joy: 

- Perhaps camelCase for `topY`, `midXl`, `midYr`, `midX`, `midY`

Reply:
Agreed! I didn't write that line but I am English...